changelog/1.40.0.md

```diff
@@ -0,0 +1,46 @@
+# 1.40.0
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.40.0.
```

Comment:
We should add something in bold that indicates all users must restart their workspaces after this update if they wish to access them.

Reply:
Er, wait.  Why do they need to restart their workspaces?